app/css/openmrs-addonmanager.css

```diff
@@ -315,13 +315,14 @@ legend.scheduler-border {
 }
 
 .install-icon, .view-icon {
-    font-size: 20px;
-    color: #1F78D5;
+    font-size: 1.05em;
+    color: #363463;
+    margin-right: 4px;
 }
 
-td {
+/* td {
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
Yes.Code should not be commented out